README.md

```diff
@@ -133,6 +164,50 @@ module "app_ecs_service" {
 
 ## Upgrade Path
 
+### 2.x.x to 3.0.0
+
+In 3.0.0 the module added support for multiple load balancer target groups. To support this change, `container_port`, `container_health_check_port` and `lb_target_group` are being replaced with `lb_target_groups`
+
+#### Without a load balancer
+
+If you are using this module without an ALB or NLB then you can remove any references to `container_port`, `container_health_check_port` and `lb_target_group` if you were doing so.
+
+#### Using with ALB or NLB target groups
+
+If you are using an NLB or NLB target groups with this module then you will need replace the values of `container_port`, `container_health_check_port` and `lb_target_group` with
+
+Below is an example of how the module would be instantiated prior to version 3.0.0
+
+```hcl
+module "app_ecs_service" {
+  source = "trussworks/ecs-service/aws"
+  ...
+  container_port              = 8443
+  container_health_check_port = 8080
+  lb_target_group             = module.alb.arn
+  ...
+}
+```
+
+In 3.0.0 the same example will look like the following
+
+```hcl
+module "app_ecs_service" {
+  source = "trussworks/ecs-service/aws"
+  ...
+  lb_target_groups =
+  [
+    {
+      container_port              = 8443
+      container_health_check_port = 8080
+      lb_target_group             = module.alb.arn
```

Comment:
Your example here does not match what the actual variable definition calls for -- one or the other needs to be fixed.